src/components/LayoutTree.tsx

```diff
@@ -247,7 +232,7 @@ const LayoutTree = () => {
         css({
           marginTop: '0.501em',
         }),
-        fauxCaretTreeProvider(indentDepth + tableDepth),
+        fauxCaretTreeProvider(Math.ceil(indent * 2)),
```

Comment:
I've moved this into `fauxCaretTreeProvider`, since this seems more like an implementation detail of the faux caret than something the `LayoutTree` needs to know about.

Also added a comment explaining the number 2, since it is not at all obvious to other developers.

See: 76f0b2e7397